Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -479,6 +479,8 @@
         "MoreInformationOnAzurePortal": "More information on Azure Portal",
         "ResponseNotValid": "Response is not in a valid format",
         "LogDeploymentName": "Deployment name is %s",
-        "ExpiredServicePrincipal": "Could not fetch access token for Azure. Verify if the Service Principal used is valid and not expired."
+        "ExpiredServicePrincipal": "Could not fetch access token for Azure. Verify if the Service Principal used is valid and not expired.",
+        "DeploymentGroupConfigurationNotSucceeded": "Deployment group configuration did not succeed on all Virtual Machines: %s",
```

Comment:
i think we can reword the message to say ... did not succeed on one or more virtual machines.

Reply:
done